its/src/test/java/com/sonar/it/scanner/msbuild/ScannerMSBuildTest.java

```diff
@@ -770,14 +770,10 @@ public void testDuplicateAnalyzersWithSameNameAreNotRemoved() throws IOException
     Path projectDir = TestUtils.projectDir(temp, "DuplicateAnalyzerReferences");
     BuildResult buildResult = runNetCoreBeginBuildAndEnd(projectDir, ScannerClassifier.NET_5);
 
-    assertThat(getComponent("DuplicateAnalyzerReferences:DuplicateAnalyzer/Program.cs"))
-      .isNotNull();
     List<Issue> issues = TestUtils.allIssues(ORCHESTRATOR);
     assertThat(issues).hasSize(2)
-      .extracting(Issue::getRule, Issue::getComponent)
-      .containsExactlyInAnyOrder(
-        tuple("csharpsquid:S1186", "DuplicateAnalyzerReferences:DuplicateAnalyzer/Program.cs"),
```

Comment:
it's still easier to read with the component ID

how can we tell from reading this test that there is an issue raised in Program and one raised in Generator?

Reply:
We cannot tell, however I am not sure it is important to tell. What is being tested here is that the build is successful and both the generator and the analyzer were run.

I can try to go back and try to assert the component ID however it is different locally vs on the CI and could not fix it easily and thought that it is not worth it investing the time.
If you think  for example a simple  comment in the code would not be enough I can go back and fight the battle with the component IDs 😄

Reply:
hm I don't understand why component IDs are different...

I am fine with not investing a lot of time in it at this point.